docs/data/charts/composition/composition.md

```diff
@@ -218,6 +219,25 @@ To add a grid, you can use the `<ChartsGrid />` component.
 
 See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information.
 
+### LegendGrid
+
+To add a a legend, you can use the `<ChartsLegend />` component.
+
+:::warning
+The Legend is an HTML element since v8.
```

Comment:
```suggestion
The legend is an HTML element since v8.
```

Reply:
The usage of capitalization come from [this writing rule](https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800)

- Legend: The legend component
- legend: The concept of legend
- `<ChartsLegend />` The piece of code